src/librustc/diagnostics.rs

```diff
@@ -335,6 +335,33 @@ This error indicates that an attempt was made to divide by zero (or take the
 remainder of a zero divisor) in a static or constant expression.
 "##,
 
+E0038: r##"
+It is not allowed to create a trait object value for a trait that is not object
+safe.  Traits are object safe only if all these conditions are met:
+
+ - the trait does not have `Self: Sized` as a bound, to enforce the abstraction
```

Comment:
Since we're using pseudo markdown in errors explanation (correct me if I'm wrong), list should start with '*' and not '-'.